src/TopDownProteomics.Benchmarks/Program.cs

```diff
@@ -39,207 +40,219 @@ private static void BenchmarkIsotopicEvelopeGeneration()
+
+                //TestLibNamespace.Northwestern.IChemicalFormula ff = new NorthwesternChemicalFormula(uwChemicalFormulas[i]);
+                //northwesternChemicalFormulas[i] = ff;
 
-                IChemicalFormula ff = new NorthwesternChemicalFormula(chemicalFormulas[i]);
-                northwesternChemicalFormulas[i] = ff;
+                var tdp_ff = new Chemistry.ChemicalFormula(chemicalFormula.Elements.Select(x => 
+                    new EntityCardinality<IElement>(elementProvider.GetElement(x.Key.AtomicNumber), x.Value)));
+                chemicalFormulas[i] = tdp_ff;
             }
 
-            Stopwatch stopwatch = new Stopwatch();
+            var stopwatch = new Stopwatch();
 
             // Time UW Madison
-
             stopwatch.Start();
-            for (int i = 2; i < 1000; i++)
+            for (int i = 2; i < MaxRunValue; i++)
             {
-                var nice = UWMadison.Chemistry.IsotopicDistribution.GetDistribution(chemicalFormulas[i], 0.2, 1E-26);
+                var nice = IsotopicDistribution.GetDistribution(uwChemicalFormulas[i], 0.2, 1E-26);
             }
             stopwatch.Stop();
 
             Console.WriteLine("Elapsed time for UofW Madison Isotopic Distribution Generation: " + stopwatch.Elapsed);
 
-            // Time Northwestern
+            // Time Northwestern OLD
+            //var mercury7_OLD = new Mercury7_OLD();
+            //stopwatch.Reset();
+            //stopwatch.Start();
+            //for (int i = 2; i < MaxRunValue; i++)
+            //{
+            //    MassSpectrometry.IIsotopicDistribution nice = mercury7_OLD.GenerateIsotopicDistribution(northwesternChemicalFormulas[i]);
+            //}
+            //stopwatch.Stop();
 
-            Mercury7 mercury7 = new Mercury7();
+            //Console.WriteLine("Elapsed time for Northwestern OLD Isotopic Distribution Generation: " + stopwatch.Elapsed);
+
+            // Time Northwestern NEW
+            var mercury7 = new Mercury7();
             stopwatch.Reset();
             stopwatch.Start();
-            for (int i = 2; i < 1000; i++)
+            for (int i = 2; i < MaxRunValue; i++)
             {
-                var nice = mercury7.GenerateIsotopicDistribution(northwesternChemicalFormulas[i]);
+                MassSpectrometry.IIsotopicDistribution nice = mercury7.GenerateIsotopicDistribution(chemicalFormulas[i]);
             }
             stopwatch.Stop();
 
-            Console.WriteLine("Elapsed time for Northwestern Isotopic Distribution Generation: " + stopwatch.Elapsed);
+            Console.WriteLine("Elapsed time for Northwestern NEW Isotopic Distribution Generation: " + stopwatch.Elapsed);
         }
-
-        #endregion Private Methods
     }
 
-    internal class NorthwesternChemicalFormula : IChemicalFormula
-    {
-        #region Private Fields
+    //internal class NorthwesternChemicalFormula : TestLibNamespace.Northwestern.IChemicalFormula
```

Comment:
Maybe just go ahead and remove this commented-out code. The change of the default to Northwestern's ChemicalFormula and such is fine by me!